vector/src/main/java/im/vector/activity/VectorUniversalLinkActivity.java

```diff
@@ -234,7 +320,7 @@ public void onUnexpectedError(Exception e) {
     public static final String EXTRA_EMAIL_VALIDATION_PARAMS = "EXTRA_EMAIL_VALIDATION_PARAMS";
 
     // Supported path
-    public static final String SUPPORTED_PATH_ACCOUNT_EMAIL_VALIDATION = "/_matrix/identity/api/v1/validate/email/submitToken";
+    public static final String EMAIL_VALIDATION_PATH_SUFFIX = "/validate/email/submitToken";
```

Comment:
Maybe update the comment too?

Reply:
ok, BTW I defined as private this member too